### ISSUE TITLE
v-model not working / two-way bind not working

### ISSUE THREAD

**Issue by realnot** [cogor/vue-formulate-datepicker#23]
I have this formulate input:

```html
<formulate-input
  v-bind="this.schema['rollout']['site']['properties']['site_acquisition_start']"
  :typeable="true"
  type="datepicker"
  label="Site Acquisition Date"
  format="dd/MM/yyyy"
  first-day-of-week="mon"
  fixed-position="bottom-left"
/>
```

When I press F5 to reload the page, the data in `form.site_acquisition_start`is loaded properly, but when I click on detail page (more fast than F5) and try to reload the record, all others form fields are loaded properly, while date input fields are not displayed (but present in form object.

Here the vue formulate form that contains all input fields

```html
  <formulate-form v-if="schema && record" v-model="form" :errors="errors" @submit="dispatch">
```

my suspect is that when record is loaded, vue-formulate is able to build `form` object through properties names, but datepicker require it's own v-model to work or acts before vue-formulate has finished to build form object which datepicker try to sync data.

<img width="1081" alt="Schermata 2021-10-29 alle 09 35 01" src="https://user-images.githubusercontent.com/2405675/139395549-30551871-b319-4d44-8a32-be6cced325dd.png">

But when I press F5

<img width="1077" alt="Schermata 2021-10-29 alle 09 43 03" src="https://user-images.githubusercontent.com/2405675/139395848-6e97cbec-fd85-41e2-a13f-95d47413bf43.png">

**Comment by realnot** [comment]
Using a `v-if` fix the issue, but I guess there's a reactivity problem.

**Comment by cogor** [comment]
@realnot Can you create sandbox?

**Comment by dovca** [comment]
@cogor I've just stumbled across the same problem with two-way binding today, here is a sandbox link: https://codesandbox.io/s/hungry-cdn-t746x

**Comment by cogor** [comment]
@dovca  @realnot Fixed in version 0.1.15.
https://codesandbox.io/s/wandering-sunset-luvw4

Thanks for your report.
